docs/tutorials/mainboard-connection.md

```diff
@@ -89,3 +104,30 @@ Pinout:
 
 Some of the `NC` pins are actually used for other purposes. For clarity and to
 avoid making mistakes they are not marked above.
+
+### Protectli VP66xx (SPI)
+
+Protectli platforms from [VP66xx series](https://eu.protectli.com/vault-6-port/)
+have header compatible with [TPM-02](https://eu.protectli.com/product/tpm02/).
+It is 2x6 pin header, with key on pin 10, and a pitch different than OrangeCrab
+(2.00 mm instead of 2.54 mm), make sure you have proper wiring. The header is
+the same as on some of the MSI boards. Gigabyte uses rotated version of this
+pinout, while it looks the same, the pinout is different.
+
+![Protectli VP66xx TPM header location](https://kb.protectli.com/wp-content/uploads/sites/9/2024/02/6600TPMHeader-1024x756.png)
+
+Pinout:
+
+![Protectli VP66xx TPM pinout](/images/pinout-2x6-key10-vert.png){ align=left }
```

Comment:
I think that the path should be [docs/images/pinout-2x6-key10-vert.png](https://github.com/Dasharo/twpm-docs/blob/e80620636fadb44a38220afb39ededd4705f1a71/docs/images/pinout-2x6-key10-vert.png) as the [images/pinout-2x10-key4-vert.png](https://github.com/Dasharo/twpm-docs/blob/e80620636fadb44a38220afb39ededd4705f1a71/images/pinout-2x6-key10-vert.png) doesn't exist and thus doesn't render properly. The same applies to the [Protectli VP46xx TPM pinout](https://github.com/Dasharo/twpm-docs/blob/e80620636fadb44a38220afb39ededd4705f1a71/docs/tutorials/mainboard-connection.md?plain=1#L90)

Reply:
It will work after publishing on https://twpm.dasharo.com, it uses different base for relative addresses.